vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthCombinedRegisterFragment.kt

```diff
@@ -55,8 +59,11 @@ import org.matrix.android.sdk.api.failure.isLoginEmailUnknown
 import org.matrix.android.sdk.api.failure.isRegistrationDisabled
 import org.matrix.android.sdk.api.failure.isUsernameInUse
 import org.matrix.android.sdk.api.failure.isWeakPassword
+import reactivecircus.flowbinding.android.widget.textChanges
 import javax.inject.Inject
 
+private const val MINIMUM_PASSWORD_LENGTH = 8
```

Comment:
My 2 cents: some homeservers may have other rules for password (length, char type usage, etc.). We may have a first error with "password must be 8 chars minimum", then later "password must be 10 chars minimum", which is quite annoying. I would avoid clients to do local check on this field and let the config server side.